packages/wallet-core/src/store/migrations/23_add_error_log.ts

```diff
@@ -0,0 +1,11 @@
+import { updateErrorReporterConfig, LiqualityErrorJSON } from '@liquality/error-parser';
+import store from '..';
+
+// Add error log to state
+export const addErrorLog = {
+  version: 23,
+  migrate: async (state: any) => {
+    updateErrorReporterConfig({ fallback: (error: LiqualityErrorJSON) => store.dispatch.logError(error) });
+    return { ...state, errorLog: [] };
```

Comment:
I don't think you need a migration for this? 
It's better to avoid migrations unless they are really necessary
This can be achieved by just creating the array if it does not exist in the mutation